src/MarketingPageAcceptanceTests.TestData/Solutions/Solution.cs

```diff
@@ -15,7 +15,7 @@ public sealed class Solution
         public string Version { get; set; }
         public string SolutionVersion => Version;
         public string SupplierId { get; set; } = "100000";
-        public int PublishedStatusId { get; set; } = 1;
+        public int PublishedStatusId { get; set; } = 3;
```

Comment:
I wouldn't do this specifically here. Leaving it as default and changing it when needed (i.e. just before creating it in the DB) would be preferable